tests/Prismic/ApiTest.php

```diff
@@ -40,7 +40,7 @@ public function setUp(): void
         unset($_COOKIE);
 
         $this->apiData = ApiData::withJsonString($this->getJsonFixture('data.json'));
-        $this->httpClient = $this->prophesize(GuzzleClient::class);
+        $this->httpClient = $this->prophesize(Client::class);
```

Comment:
Is there a reason why you cannot mock `GuzzleHttp\ClientInterface` - Just a note - I'm assuming that where relevant the interface is consumed and it's safer to mock interfaces over implementations.

Reply:
Done in [132244b](https://github.com/prismicio-community/php-kit/pull/199/commits/132244bf4c513c7d35a20ea15b55d33e3a44d311)